src/main/java/ch/njol/skript/SkriptCommand.java

```diff
@@ -124,15 +142,24 @@ public boolean onCommand(CommandSender sender, Command command, String label, St
 		if (!SKRIPT_COMMAND_HELP.test(sender, args))
 			return true;
 
+		List<CommandSender> recipients = new ArrayList<>();
```

Comment:
This could be a Set, which would mean you don't have to do the .equals check for each player.